app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser_dependencies.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export type {
+    AppCall,
+    AppBinding,
+    AppField,
+    AppSelectOption,
+    AppCallResponse,
+    AppContext,
+    AppForm,
+    AutocompleteElement,
+    AutocompleteDynamicSelect,
+    AutocompleteStaticSelect,
+    AutocompleteUserSelect,
+    AutocompleteChannelSelect,
+    AppLookupCallValues,
+} from '@mm-redux/types/apps';
```

Comment:
The point of this file is so that `app_command_parser.ts` can be identical on webapp and mobile. This way any fixes are much easier to propagate across the platforms. This file allows that `app_command_parser.ts` to import these things on webapp and mobile, agnostic to which platform it is running on.